Makefile

```diff
@@ -32,6 +31,11 @@ frontend:
 	docker build --build-arg CONFIGURATION=local -t t4c/client/frontend -t $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend frontend
 	docker push $(LOCAL_REGISTRY_NAME):$(REGISTRY_PORT)/capella/collab/frontend
 
+capella: capella-5.0.0 capella-5.2.0 capella-6.0.0
```

Comment:
I'd not add it as default - The whole setup is too much for most notebooks. Maybe an Make variable for the CAPELLA_VERSION(S)?

Reply:
I added a Make variable with to-be built versions.